src/us/kbase/workspace/test/kbase/HandleAndShockTest.java

```diff
@@ -417,84 +427,62 @@ public void basicHandleTest() throws Exception {
 		 * object where they don't own the shock nodes, even with all other
 		 * privileges
 		 */
-		BasicShockClient bsc = new BasicShockClient(
-				new URL("http://localhost:" + SHOCK.getServerPort()), CLIENT1.getToken());
-		@SuppressWarnings("unused")
-		ShockACL acl = bsc.addToNodeAcl(new ShockNodeId(h1.getId()),
-				Arrays.asList(USER2), ShockACLType.ALL);
+		bsc.addToNodeAcl(new ShockNodeId(shock_id), Arrays.asList(USER2), ShockACLType.ALL);
 		String workspace2 = "basichandle2";
 		CLIENT2.createWorkspace(new CreateWorkspaceParams().withWorkspace(workspace2));
 		try {
 			CLIENT2.saveObjects(new SaveObjectsParams().withWorkspace(workspace2)
 					.withObjects(Arrays.asList(
 							new ObjectSaveData().withData(new UObject(handleobj)).withName("foo2")
 							.withType(HANDLE_TYPE))));
-			fail("saved object with bad handle");
-		} catch (ServerException e) {
```

Comment:
This should not have changed. This operation should fail and the assert below should succeed.

Reply:
👍